spec/state_of_the_nation_spec.rb

```diff
@@ -64,17 +59,6 @@ class Country < ActiveRecord::Base
     m.create_table :countries
   end
 
-  after :all do
-    President = @existing_president_constant
-    Country = @existing_country_constant
-    Senator = @existing_senator_constant
-
-    m = ActiveRecord::Migration.new
-    m.drop_table :presidents
-    m.drop_table :senators
-    m.drop_table :countries
```

Comment:
I take it we don't need this because the top-level test transaction rollback will remove them?

I would've thought that was done at a point equivalent to `each`, or are there multiple levels of nested transactions?

Reply:
> I take it we don't need this because the top-level test transaction rollback will remove them?

We could drop the tables after each run but it feels unnecessary as they are reused for each run but maybe as we only use them in this test file it might be better to remove them after.

> I would've thought that was done at a point equivalent to each, or are there multiple levels of nested transactions?

`DatabaseCleaner` truncates the tables after each run but doesn't drop the tables.